### PR TITLE
* remove c dependency and use the built in intrinsic wrappers

### PR DESCRIPTION
**Pull request by adamnemecek**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,6 +3,6 @@ import PackageDescription
 let package = Package(
     name: "SwiftBitset",
     targets: [
```
Comment (lemire) [inline Package.swift]: I think we need to add "swiftLanguageVersions: [4]".

Comment (lemire) [reply Package.swift]: https://github.com/apple/swift-evolution/blob/master/proposals/0151-package-manager-swift-language-compatibility-version.md

File: Sources/Bitset/Bitset.swift [hunk 2]
```diff
@@ -225,22 +211,22 @@ public final class Bitset: Sequence, Equatable, CustomStringConvertible,
 
   // compute the intersection (in place) with another bitset
   public func intersection(_ other: Bitset) {
-    let mincount = other.wordcount < self.wordcount ? other.wordcount : self.wordcount
+    let mincount = Swift.min(self.wordcount, other.wordcount)
```
Comment (lemire) [inline Sources/Bitset/Bitset.swift:214]: Any particular reason for using "Swift.min" as opposed to just "min"?

**Review by lemire — COMMENTED** [review]
Please consider the couple of comments I made.

**Comment by lemire** [comment]
Great stuff. We will need an AUTHORS file now!

I'll be happy to merge this, just look at my questions first.

File: Sources/Bitset/Bitset.swift (hunk 2 above)
Comment (adamnemecek) [reply Sources/Bitset/Bitset.swift:214]: the compiler complains because min refers to the the min() method.

**Comment by adamnemecek** [comment]
By the way, I think that the way forward would be to make this a struct that conforms to the SetAlgebra protocol. I can make the changes but I'm not super sure how much of an impact this might have on perf. I can't imagine it would be significant.

**Comment by lemire** [comment]
Checking performance: 

```
$ git clone https://github.com/lemire/SwiftBitsetBenchmark.git
$ cd SwiftBitsetBenchmark
$ swift build  -Xcc -march=native  --configuration release

$ .build/release/SwiftBitsetBenchmark
testAddPerformance  10.693318  ms
testIndexSetAddPerformance  231.737616  ms

testCountPerformance  0.617098  ms
testIndexSetCountPerformance  0.007483  ms

testIteratorPerformance  5.503873  ms
testIndexSetIteratorPerformance  234.289692  ms

testIntersectionPerformance  3.157883  ms
testIndexSetIntersectionPerformance  2774.959423  ms

$ (do some editing)
$ git diff
diff --git a/Package.swift b/Package.swift
index 24306d9..47b7011 100644
--- a/Package.swift
+++ b/Package.swift
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "SwiftBitsetBenchmark",
     dependencies: [
-   .Package(url: "https://github.com/lemire/SwiftBitset.git",  majorVersion: 0),
+   .Package(url: "https://github.com/adamnemecek/SwiftBitset",  majorVersion: 0),
    .Package(url: "https://github.com/lemire/Swimsuit.git",  majorVersion: 0)
     ]
 )

$ rm -r -f Package.resolved Packages
$ swift build  -Xcc -march=native  --configuration release

$ .build/release/SwiftBitsetBenchmark
testAddPerformance  10.617662  ms
testIndexSetAddPerformance  231.558306  ms

testCountPerformance  0.611426  ms
testIndexSetCountPerformance  0.007426  ms

testIteratorPerformance  5.348081  ms
testIndexSetIteratorPerformance  234.289538  ms

testIntersectionPerformance  3.126475  ms
testIndexSetIntersectionPerformance  2724.773162  ms
```

So it is unchanged.

I'll merge this. If you want to make further changes and you are concerned about performance, we can run the same benchmark and see what happens. Of course, my benchmark is not exhaustive, but we could extend it if needed.

**Comment by lemire** [comment]
See https://github.com/lemire/SwiftBitset/blob/master/AUTHORS

**Pull request merged**